src/components/ProposalsList.vue

```diff
@@ -0,0 +1,46 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+import type { Proposal as ProposalType } from '@/types';
+
+const props = defineProps<{
+  title: string;
+  loading?: boolean;
+  limit?: number | false;
+  proposals: ProposalType[];
+  route?: {
+    name: string;
+    linkTitle: string;
+  };
+}>();
+
+const limit = computed(() => {
+  if (props.limit === false) return Infinity;
+
+  return props.limit || 3;
```

Comment:
It seem that the limit of 3 doesnt work for me, I can see 4 closed proposals on this space:
https://bafybeibh4naq5yi67sa43jmlpoojvh646f5oyrsc6fgqd7h6lcpy7i4gd4.on.fleek.co/#/0x04b7cff71219e275676e0ca23579f41b99dd1d1bd01adc7d7f1bc917d448e57d